### PR TITLE
Use same styling for statuses in email as on web

### PR DESCRIPTION
**Pull request by c960657**
The notification emails containing statuses are rendered with a different styling than on the web.

- White-space are collapsed, so ASCII art or spaces used for indentation will not work.
- There is no margin between paragraphs, so two consecutive newlines look like a single newline.
- Links are not displayed in their compact format (leading `https://www` is stripped and long path is truncated with ellipsis).
- A very long word will make the container grow horizontally (in Mail.app on macOS).

This PR does address the difference in font family and size. I assume that is intentional.

### CODE REVIEW

**Review by ClearlyClaire — APPROVED** [review]
Looks ok at a glance but I'd like another set of eyes, especially since both the design and its implementation have been carried out by people external from the core team with a lot of expertise in the field.

**Comment by renchap** [comment]
Pinging @hteumeuleu if you can have a look, as you wrote this original CSS?

**Comment by hteumeuleu** [comment]
Just had a quick look and this should be fine.

**Comment by mjankowski** [comment]
I think this might have had unintended consequences on the welcome mailer...

Currently in main:

Screenshot 2024-09-27 at 13 16 55

If I comment out the `table-layout` change from this PR:

Screenshot 2024-09-27 at 13 17 04

**Comment by c960657** [comment]
Fix coming up (after dinner).

**Comment by c960657** [comment]
Follow-up PR created: #32132

@mjankowski I have fixed the issue with the collapsed banner image. Did you notice other problems? I might have missed something while comparing the two screenshots.

The text wrapping in the buttons look ugly, so I fixed that as well, even though this is not a new problem. On my machine, the font is a bit more compact than that in your screenshot, so here the “Personalize” text fits within the default button width.

**Comment by mjankowski** [comment]
> Follow-up PR created: #32132
> 
> @mjankowski I have fixed the issue with the collapsed banner image. Did you notice other problems? I might have missed something while comparing the two screenshots.

That's all I noticed -- the vertical white line and missing instance card ... although I wasn't exhaustive across all mailers, just noticed that in welcome while looking at something unrelated.

> The text wrapping in the buttons look ugly, so I fixed that as well, even though this is not a new problem. On my machine, the font is a bit more compact than that in your screenshot, so here the “Personalize” text fits within the default button width.

🥳

**Pull request merged**
